modules/ROOT/pages/sso-use-case-2.adoc

```diff
@@ -9,31 +11,38 @@ The instructions in this case are using Okta as an example identity provider how
 
 
 == Before You Begin
-Ensure you have the following setup:
+Ensure you have the following setup and context:
 
 * An identity provider for Anypoint Platform is already configured.
 * The user or the identity of the group of the users who have access to the application have been created in the identity provider. 
+* Understand the required roles, permissions, and information that API Experience Hub requires to configure custom user attributes and claims for the identity provider.
++
+See xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin].
```

Comment:
Does this work with the extra space?

Reply:
Thanks for catching that. I will removed the extra space.